cli/lib/tasks/download.js

```diff
@@ -54,6 +54,20 @@ const getBaseUrl = () => {
   return defaultBaseUrl
 }
 
+const getCA = () => {
+  let ca
+
+  if (util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CA')) {
+    ca = [util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CA')]
+  }
+
+  if (util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CAFILE')) {
+    ca = (ca || []).concat(fs.readFileSync(util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CAFILE')).toString())
```

Comment:
There needs to be error handling around this file read, so the CLI doesn't just crash with ENOENT if the file doesn't exist. Also, use the promise API `fs.readFile` instead of the sync method.

Reply:
Will do the change a bit later tomorrow.